mcsema/Arch/ABI.cpp

```diff
@@ -317,9 +317,12 @@ static uint64_t DefaultUsedStackBytes(llvm::CallingConv::ID cc) {
   }
 }
 
-static const char *IntReturnValVar(llvm::CallingConv::ID cc) {
-  if (llvm::CallingConv::X86_64_SysV == cc ||
-      llvm::CallingConv::Win64 == cc) {
+static const char *IntReturnValVar(llvm::CallingConv::ID cc,
+                                   size_t index) {
+  if (llvm::CallingConv::X86_64_SysV == cc) {
+    static const char *regs[2] = {"RAX", "RDX"};
+    return regs[index];
+  } else if (llvm::CallingConv::Win64 == cc) {
     return "RAX";
```

Comment:
Perhaps lets break the `else if` out into its own `if`, then have a `LOG_IF(FATAL, index)` with a message about multiple return values not being supported for the specific calling convention.